README.md

```diff
@@ -2,16 +2,22 @@
 
 A collection of copy-pastable code example snippets demonstrating the various ways to use the Vault client libraries for various languages to authenticate and retrieve secrets.
 
-## How To Use 
+## Currently Supported Languages
 
-Find the relevant file inside the designated directory for the language of your choice, and paste the example code into your application. (This repo is not intended to be "run" as a standalone application.)
+- Go
+  - Uses official library [HashiCorp Vault](https://pkg.go.dev/github.com/hashicorp/vault/api)
+  - Provided examples:
+    - Auth Methods (AppRole, AWS, Azure, GCP, Kubernetes)
+    - Token Renewal
+- C#
+  - Uses community-maintained library [VaultSharp](https://github.com/rajanadar/VaultSharp)
+  - Provided examples:
+    - Auth Methods (AppRole, AWS, Azure, GCP, Kubernetes)
 
-### Dotnet
-These examples use the community maintained library: [VaultSharp](https://github.com/rajanadar/VaultSharp)
+## How To Use
 
-### Go
-These examples use the HashiCorp maintained client library: [HashiCorp Vault](https://pkg.go.dev/github.com/hashicorp/vault/api)
+Find the relevant file inside the designated directory for the language of your choice, and paste the example code into your application. (This repo is not intended to be "run" as a standalone application.)
```

Comment:
You may want to run prettier on this, for example this default:

```sh
prettier --print-width=80 --prose-wrap=always -w README.md
```

Reply:
Thanks, finally prompted me to figure out why my Prettier extension wasn't working